src/elm/View/Modal.elm

```diff
@@ -0,0 +1,59 @@
+module View.Modal exposing (..)
+
+import Html exposing (Html, button, div, text)
+import Html.Attributes exposing (class)
+import Html.Events exposing (onClick, stopPropagationOn)
+import Icons
+import Json.Decode as Decode
+
+
+type alias Config msg =
+    { content : Html msg
+    , closeMsg : msg
+    , ignoreMsg : msg
+    }
+
+
+type Visibility
+    = Hidden
+    | Shown
+
+
+hidden : Visibility
+hidden =
+    Hidden
+
+
+shown : Visibility
+shown =
+    Shown
+
+
+view : Visibility -> Config msg -> Html msg
+view status cfg =
+    case status of
+        Shown ->
+            div
+                [ class "modal container fade-in"
+                , stopPropagationOn "click" (Decode.succeed ( cfg.ignoreMsg, True ))
+                ]
+                [ div
+                    [ class "modal-bg"
+                    , onClick cfg.closeMsg
+                    ]
+                    []
+                , div
+                    [ class "modal-content overflow-auto" ]
```

Comment:
maybe this `overflow-auto` is messing up the content when the error message appears, check it out:

<img width="948" alt="Screen Shot 2020-07-15 at 14 23 05" src="https://user-images.githubusercontent.com/1082127/87575813-dcd46800-c6a6-11ea-8d87-d70917f93adc.png">

Reply:
oh I forgot you created an issue for this. #292 should address this!